tests/lib/icmsg/testcase.yaml

```diff
@@ -0,0 +1,4 @@
+tests:
+  libraries.icmsg:
+    integration_platforms:
+      - nrf5340dk_nrf5340_cpuapp
```

Comment:
Please add some meaningful way of filtering this test for twister. Integration_platforms only works for PRs. In case of a nightly build twister tries to build "everything on everything" and this test will be catch in scope for every single platform (~400) in the tree and probably fail https://docs.zephyrproject.org/latest/guides/test/twister.html#test-cases

Reply:
This test should be platform agnostic so it probably should be run on everything possible.